.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,48 @@
+<!--
+PLEASE HELP US PROCESS GITHUB ISSUES FASTER BY PROVIDING THE FOLLOWING INFORMATION.
+
+NOTE: IF YOU HAVE A *QUESTION*, PLEASE POST IT ON THE OKTA DEVELOPER FORUM (https://devforum.okta.com) INSTEAD. ISSUES IN THIS REPOSITORY ARE RESERVED FOR BUG REPORTS AND FEATURE REQUESTS.
+-->
+
+## I'm submitting this issue for the package(s):
+- [ ] jwt-verifier
+- [ ] okta-angular
+- [ ] oidc-middleware
+- [ ] okta-react
+- [ ] okta-react-native
+- [ ] okta-react-native
+- [ ] okta-vue
+
+## I'm submitting a:
+- [ ] Regression (a behavior that used to work and stopped working in a new release)
+- [ ] Bug report  <!-- Please search GitHub for a similar issue or PR before submitting -->
+- [ ] Performance issue
+- [ ] Feature request
+- [ ] Documentation issue or request
+- [ ] Other... Please describe:
+
+## Current behavior
+<!-- Describe how the issue manifests. -->
+
+
+## Expected behavior
+<!-- Describe what the desired behavior would be. -->
+
+
+## Minimal reproduction of the problem with instructions
+<!--
+For bug reports please provide the *STEPS TO REPRODUCE* and if possible a *MINIMAL DEMO* of the problem.
```

Comment:
We could suggest that they use one of our sample applications to build a reproduction demo